packages/amplify-category-custom/src/index.ts

```diff
@@ -18,6 +18,8 @@ export async function executeAmplifyCommand(context: $TSContext) {
 
   const commandModule = await import(commandPath);
 
+  pathManager.getAmplifyMetaFilePath();
```

Comment:
This should work, but it's confusing to rely on the error throwing side effect of `pathManager.getAmplifyMetaFilePath();` when we don't need the meta file path.

A better solution would be to add a stateManager method like `isProjectInitialized()` and call that here. That will make it easier to provide a useful error message, instead of throwing an error.

Reply:
Thank you for the feedback. I just have some questions: should I still call `pathManager.getAmplifyMetaFilePath()` in the `isProjectInitialized()` to validate that the project is initialized or is there a better way to do the validation? and, should the new method throw an error or just the error message?

Reply:
On second thought, I think we can just use an existing stateManager method and leave a comment. I think implementing `isProjectInitialized()` robustly is out of scope for this issue.

```ts
// Check if project has been initialized
if (!stateManager.metaFileExists()) {
  printer.error('Could not find the amplfiy-meta.json file. Make sure your project is initialized in the cloud.');
}
```